progect-management-app/src/app/components/dashboard/column-form/column-form.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, EventEmitter, Output } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+
+@Component({
+  selector: 'app-column-form',
+  templateUrl: './column-form.component.html',
+  styleUrls: ['./column-form.component.scss']
```

Comment:
According to best practices, it better to use onPush strategy ( changeDetection: ChangeDetectionStrategy.OnPush) for each component. That helps you to prevent performance issue with application.